### PR TITLE
Ch s/v2/prices component

### PR DESCRIPTION
**Pull request by SuppressorButtons**
Was fully implemented prices.component. Repaired source and links for images(v2/src/assets/images)

### CODE REVIEW

File: v2/src/app/components/prices/prices.component.ts [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+    private pricesService: PricesService) {
+
+  }
+
+  ngOnInit() {
+    this.pricesService.getPrices().subscribe(
+      (res: any) => {
+        this.prices = res.map((price)=>{
+          return {
+            // ...price, // WTF ????? //todo Should ask Aleksandr
```
Comment (AlexOrd) [inline v2/src/app/components/prices/prices.component.ts]: TODO: Investigate price visualization.

File: v2/src/app/services/prices.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+export class PricesService {
+  private pricesUrl = '/api/getallprices';
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  getPrices() {
+    return this.http.get(this.pricesUrl)
+    // return [{ marker: { lat: 49.229065, long: 28.425729} }]
```
Comment (AlexOrd) [inline v2/src/app/services/prices.service.ts:18]: Unnecessary comments.

**Review by AlexOrd — CHANGES_REQUESTED** [review]

**Review by SuppressorButtons — COMMENTED** [review]
Problem with npm settings

**Comment by ghost** [comment]
Please merged this .... I waiting this ... I suhd use some portion of this component and application settings.

**Pull request merged**